openwisp_utils/tests.py

```diff
@@ -76,3 +77,70 @@ def stopTestRun(self):
 class TimeLoggingTestRunner(DiscoverRunner):
     def get_resultclass(self):
         return TimeLoggingTestResult
+
+
+class redirect_stdout(object):
+    def __init__(self, stdout, **kwargs):
+        """Init and set vars"""
+        self.kwargs = kwargs
+        self.stdout = stdout
+
+    def __call__(self, function):
+        def wrapped_function(*args, **kwargs):
+
+            sys.stdout = self.stdout
+
+            try:
+                function(*args, self.stdout, **kwargs)
+
+            finally:
+                self.stdout.close()
+                sys.stdout = sys.__stdout__
+
+        return wrapped_function
+
+
+class redirect_stderr(object):
+    def __init__(self, stderr, **kwargs):
+        """Init and set vars"""
+        self.kwargs = kwargs
+        self.stderr = stderr
+
+    def __call__(self, function):
+        def wrapped_function(*args, **kwargs):
+
+            sys.stderr = self.stderr
+
+            try:
+                function(*args, self.stderr ** kwargs)
+
+            finally:
+                self.stderr.close()
+                sys.stderr = sys.__stderr__
+
+        return wrapped_function
+
+
+class redirect_any_output(object):
```

Comment:
Could you please document these classes in `README.md`? Please include how to use them.